packages/react-components/react-migration-v0-v9/src/components/Button/ButtonMigration.mixins.test.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { GriffelStyle, shorthands } from '@fluentui/react-components';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { buttonMigrationStyles } from './ButtonMigration.mixins';
+
+const testMixin = (mixin: GriffelStyle | undefined, expectedStyle: GriffelStyle | undefined) => {
+  const name = expectedStyle ? JSON.stringify(expectedStyle) : 'empty';
+  test(name, () => {
+    const result = { ...mixin };
+    expect(result).toEqual(expectedStyle || {});
+  });
+};
+
+describe('ButtonMigration.mixins', () => {
+  describe('classNames check', () => {
+    test('v9: iconRegularClassName', () => {
+      expect(iconRegularClassName).toEqual('fui-Icon-regular');
+    });
+    test('v9: iconFilledClassName', () => {
+      expect(iconFilledClassName).toEqual('fui-Icon-filled');
+    });
+  });
+  describe('v0Icon', () => {
+    const styles = {
+      '& .ui-icon__outline': {
+        display: 'block',
+      },
+      '& .ui-icon__filled': {
+        display: 'none',
+      },
+      '&:hover': {
+        '& .ui-icon__outline': {
+          display: 'none',
+        },
+        '& .ui-icon__filled': {
+          display: 'block',
+        },
+      },
+    };
+    testMixin(buttonMigrationStyles.v0Icon(), styles);
```

Comment:
Can't you use just `buttonMigrationStyles.v0Icon().toMatchInlineSnapshot()`?

Reply:
Done